Userland/Applications/TextEditor/MainWidget.h

```diff
@@ -16,14 +16,39 @@
 #include <LibGUI/Window.h>
 #include <LibWeb/Forward.h>
 
+#include <FileSystemAccessServer/ClientConnection.h>
+#include <FileSystemAccessServer/FileSystemAccessClientEndpoint.h>
+#include <FileSystemAccessServer/FileSystemAccessServerEndpoint.h>
+#include <LibIPC/ServerConnection.h>
+
 namespace TextEditor {
 
+class FileSystemAccessClient final
+    : public IPC::ServerConnection<FileSystemAccessClientEndpoint, FileSystemAccessServerEndpoint>
+    , public FileSystemAccessClientEndpoint {
+    C_OBJECT(FileSystemAccessClient)
+
+public:
+    virtual void die() override
+    {
+    }
+
+private:
+    explicit FileSystemAccessClient()
+        : IPC::ServerConnection<FileSystemAccessClientEndpoint, FileSystemAccessServerEndpoint>(*this, "/tmp/portal/fileaccess")
+    {
+    }
+    virtual void dummy() override
```

Comment:
I think the dummy methods should no longer be needed since #8221 has been merged.